src/platform_impl/windows/event_loop.rs

```diff
@@ -320,36 +360,34 @@ fn get_wait_thread_id() -> DWORD {
             "this shouldn't be possible. please open an issue with Winit. error code: {}",
             result
         );
-        msg.lParam as DWORD
+        msg.lParam as u32
     }
 }
 
 lazy_static! {
-    static ref WAIT_PERIOD_MIN: Option<UINT> = unsafe {
-        let mut caps = mmsystem::TIMECAPS {
+    static ref WAIT_PERIOD_MIN: Option<u32> = unsafe {
+        let mut caps = TIMECAPS {
             wPeriodMin: 0,
             wPeriodMax: 0,
         };
-        if timeapi::timeGetDevCaps(&mut caps, mem::size_of::<mmsystem::TIMECAPS>() as _)
-            == mmsystem::TIMERR_NOERROR
-        {
+        if timeGetDevCaps(&mut caps, mem::size_of::<TIMECAPS>() as u32) == TIMERR_NOERROR {
             Some(caps.wPeriodMin)
         } else {
             None
         }
     };
 }
 
-fn wait_thread(parent_thread_id: DWORD, msg_window_id: HWND) {
+fn wait_thread(parent_thread_id: u32, msg_window_id: HWND) {
     unsafe {
-        let mut msg: winuser::MSG;
+        let mut msg: MSG;
 
-        let cur_thread_id = processthreadsapi::GetCurrentThreadId();
-        winuser::PostThreadMessageW(
+        let cur_thread_id = GetCurrentThreadId();
+        PostThreadMessageW(
             parent_thread_id,
             *SEND_WAIT_THREAD_ID_MSG_ID,
             0,
-            cur_thread_id as LPARAM,
+            cur_thread_id as isize,
```

Comment:
Nit: Use `windows_sys::Win32::Foundation::LPARAM` (mostly to keep the diff smaller)